jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -128,7 +128,17 @@ ecma_object_is_constructor (ecma_object_t *obj_p) /**< ecma object */
     return (!ecma_get_object_is_builtin (obj_p) || !ecma_builtin_function_is_routine (obj_p));
   }
 
-  return (type >= ECMA_OBJECT_TYPE_BOUND_FUNCTION);
+  if (type == ECMA_OBJECT_TYPE_BOUND_FUNCTION)
+  {
+    ecma_bound_function_t *bound_func_p = (ecma_bound_function_t *) obj_p;
+
+    ecma_object_t *func_obj_p = ECMA_GET_NON_NULL_POINTER_FROM_POINTER_TAG (
+        ecma_object_t, bound_func_p->header.u.bound_function.target_function);
```

Comment:
This indentation is a bit strange.

How about this?

```c
ecma_object_t *func_obj_p;
func_obj_p = ECMA_GET_NON_NULL_POINTER_FROM_POINTER_TAG (ecma_object_t,
                                                         bound_func_p->header.u.bound_function.target_function);
```

Reply:
It was proposed by clang-format.

Unfortunately the formatting you showed, does not pass vera check:
```
./jerry-core/ecma/operations/ecma-function-object.c:136: error: line is longer than 120 characters
```
It has 132 characters.

Reply:
Done.